catalogue/webapp/pages/work.js

```diff
@@ -29,12 +181,142 @@ const WorkPage = ({ work }: Props) => {
+                    <MoreInfoLink name='View Wellcome Library catalogue record' url={encoreLink} />
+                  </div>
+                }
+              </div>
+
+              <WorkDrawer data={[{
+                headingText: 'License information',
+                text: getLicenseInfo(work.thumbnail.license.licenseType).humanReadableText
+              }, {
+                headingText: 'Credit',
+                text: attribution
+              }]} />
+            </div>
+
+            <div className={classNames([
+              grid({s: 12, m: 10, shiftM: 1, l: 5, xl: 5}),
+              spacing({s: 1}, {margin: ['top']})
+            ])}>
+              <h2 className={classNames([
+                font({s: 'HNM4', m: 'HNM3'}),
+                spacing({s: 0}, {margin: ['top']}),
+                spacing({s: 2}, {margin: ['bottom']})
+              ])}>
+                Download
+              </h2>
+
+              {/* TODO: the download links once this is in
+              https://github.com/wellcometrust/wellcomecollection.org/pull/2164/files#diff-f9d8c53a2dbf55f0c9190e6fbd99e45cR21 */}
+              {/* the small one is 760 */}
+              <a
+                className={classNames([
+                  spacing({s: 2}, {margin: ['bottom']}),
+                  font({s: 'HNM5', m: 'HNM4'}),
+                  'plain-link font-green font-hover-turquoise flex flex--v-center'
+                ])}
+                href={`/download?uri=${encodeURIComponent('/link/to/image')}`}
+                download={`${work.id}.jpg`}
+                rel='noopener noreferrer'
+                data-track-event={JSON.stringify({
+                  'category': 'component',
+                  'action': 'download-button:click',
+                  'label': `id: work.id , size:original, title:${work.title.substring(50)}`
+                })}>
+                <Icon name='download' extraClasses='icon--green' />
+                <div className={spacing({s: 1}, {margin: ['left']})}>
+                  Download original
+                </div>
+              </a>
+
+              <a
+                className={classNames([
+                  spacing({s: 4}, {margin: ['bottom']}),
+                  font({s: 'HNM5', m: 'HNM4'}),
+                  'plain-link font-green font-hover-turquoise flex flex--v-center'
+                ])}
+                href={`/download?uri=${encodeURIComponent('/link/to/image')}`}
+                download={`${work.id}.jpg`}
+                rel='noopener noreferrer'
+                data-track-event={JSON.stringify({
+                  'category': 'component',
+                  'action': 'download-button:click',
+                  'label': `id: work.id , size:760, title:${work.title.substring(50)}`
+                })}>
+                <Icon name='download' extraClasses='icon--green' />
+                <div className={spacing({s: 1}, {margin: ['left']})}>
+                  Download small (760px)
```

Comment:
I was about to pull you up on a linting error before I realised `small (760px)` isn't a function.